src/main/java/io/xpring/xrpl/javascript/JavaScriptUtils.java

```diff
@@ -124,4 +124,21 @@ public String toTransactionHash(String transactionBlobHex) {
     Value hash = transactionBlobToTransactionHashFunction.execute(transactionBlobHex);
     return hash.isNull() ? null : hash.toString();
   }
+
+  /**
+   * Convert the given byte array to a hexadecimal string.
+   *
+   * @param bytes An array of bytes
+   * @return An encoded hexadecimal string.
```

Comment:
I like to include what the @param and @return represent so there is no confusion.